src/components/collaboration/CollaborativeNotes.tsx

```
@@ -0,0 +1,253 @@
+
+    onDocumentChange({
+      ...document,
+      content: newContent,
+      version: document.version + 1,
+      updatedAt: new Date().toISOString(),
+      lastEditor: currentUser.name,
+    });
+  };
+
+  const handleAddTag = (e: React.KeyboardEvent) => {
+    if (e.key === 'Enter' && tagInput.trim()) {
```

ESLint

[error] Replace `'Enter'` with `"Enter"`

(prettier/prettier)